internal/provider/resource_project.go

```diff
@@ -663,8 +1049,10 @@ func resourceWizProjectDelete(ctx context.Context, d *schema.ResourceData, m int
 	// populate the graphql variables
 	vars := &vendor.UpdateProjectInput{}
 	vars.ID = d.Id()
-	vars.Patch.Name = d.Get("slug").(string)
-	vars.Patch.Archived = utils.ConvertBoolToPointer(d.Get("archived").(bool))
+	vars.Override.Name = d.Get("slug").(string)
+	vars.Override.Slug = d.Get("slug").(string)
+	d.Set("archived", true)
```

Comment:
@gramsa49  please note the setting of `archived` as `true` in state before populating the vars for request

Reply:
That works!